home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/CooperationDto.java

```diff
@@ -1,26 +0,0 @@
-package com.softserveinc.ita.homeproject.homeservice.dto;
```

Comment:
I think that dto package should be organized like entity package. MailDto can be added to package “invitation” or left in root.